README.md

```diff
@@ -124,7 +139,9 @@ Your query here
 6. The name of the office with its number of employees ordered in a desc.
 
 ```
-Your query here
+SELECT o.name, count(e.office_id) as count
+FROM offices o LEFT JOIN employees e ON o.id = e.office_id 
```

Comment:
Considering you must take the mutual records between employees and offices, why do you use a `LEFT JOIN` ?

Reply:
Thank you for the observation, Sergio. Yes, I used to use the LEFT JOIN to see null records, however on this occasion was a better choice the INNER JOIN to have the matching records table.